lib/structures/VoiceState.ts

```diff
@@ -96,7 +96,7 @@ export default class VoiceState extends Base {
 
     /** The channel the user is connected to. */
     get channel(): VoiceChannel | StageChannel | null | undefined {
-        if (this.channelID !== null && this._cachedChannel !== null) {
+        if (this.channelID !== null && this._cachedChannel === null) {
```

Comment:
this will work for the first get then return null on any subsequent attempts afterwards.

Reply:
I planned on rewriting these getters to use `variable ??= value` instead of `variable ?? (varibable = value)`

Reply:
You can actually just completely remove ` && this._cachedChannel === null`